core/infra/config/iroha_config_with_json.hpp

```diff
@@ -17,32 +17,32 @@ limitations under the License.
 #ifndef IROHA_CONFIG_WITH_JSON_HPP
 #define IROHA_CONFIG_WITH_JSON_HPP
 
-#include "iroha_config.hpp"
+#include "abstract_config_manager.hpp"
+
+using ACM = config::AbstractConfigManager;
```

Comment:
Abbreviation is really confusing, full name will be a way more readable I think.